src/github.com/travis-ci/jupiter-brain/vsphere.go

```diff
@@ -301,10 +302,27 @@ func (i *vSphereInstanceManager) createClient(ctx context.Context) (*govmomi.Cli
 	client.Client.RoundTripper = &soapBreakerRoundTripper{
 		rt: client.Client.RoundTripper,
 		cb: gobreaker.NewCircuitBreaker(gobreaker.Settings{
-			Name: "vSphere govmomi",
+			Name:     "vSphere govmomi",
+			Interval: 10 * time.Minute,
 			ReadyToTrip: func(counts gobreaker.Counts) bool {
 				failureRatio := float64(counts.TotalFailures) / float64(counts.Requests)
-				return counts.Requests >= 3 && failureRatio >= 0.6
+				return counts.Requests >= 5 && failureRatio >= 0.6
```

Comment:
also, is it "60% of requests over 10 minutes" or is it more just 
"if (failed/total) > .6" every time that a request is made?